### ISSUE TITLE
Unity without support building for IOS

### ISSUE THREAD

**Issue by dimmduh** [yandexmobile/metrica-plugin-unity#7]
If Unity does not support building for IOS (You can select it while installing), after import appmetrica there are some fatal errors in console (you can build or start).

I added:
#if UNITY_IPHONE || UNITY_STANDALONE_OSX
to PostprocessBuildPlayerAppMetrica.cs and it helps
See https://gist.github.com/dimmduh/f2d33246aedb1ca398a3901c63ee39c6
I guess this problem can be when Unity can build for IOS and does not support build for Android

**Comment by bamx23** [comment]
You're right. Temporary solution is to cover all `PostprocessBuildPlayerAppMetrica.cs` with:
```csharp
#if UNITY_IPHONE 

// Actual PostprocessBuildPlayerAppMetrica.cs content

#endif
```
